packages/state/src/RollupState.ts

```diff
@@ -57,7 +57,7 @@ export class RollupState {
           throw new Error('Invalid update count')
         }
         const funding =
-          update.fundingTimestamp !== 0n
+          Number(update.fundingTimestamp) !== 0
```

Comment:
Maybe `Timestamp(0)` would be cleaner here?

Reply:
You're right - done.